contentcuration/contentcuration/migrations/0147_create_embeddings_table.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.19 on 2023-11-01 12:00
+from django.db import migrations
+from django.db import models
+from pgvector.django import VectorExtension
+from pgvector.django import VectorField
+
+import contentcuration.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contentcuration', '0146_drop_taskresult_fields'),
+    ]
+
+    operations = [
+        VectorExtension(),
```

Comment:
I'm quite certain we cannot have this in the django migration, as the db user in our production deployments doesn't have the appropriate permissions to enable extensions. So that said, in production it will have to be a manual step. In development, I see a few options:
1) Use the postgres' image SQL init to run the SQL that enables it
2) Add it to the `devsetup` management command

Also because of this, we should update the [Host setup guide](https://github.com/learningequality/studio/blob/unstable/docs/host_services_setup.md) depending on which options you move forward with.

Reply:
Ohh, then I think using the `devsetup` management command for this will be better. Because the postgres' image SQL init assumes that the developer must be using our docker setup whereas `devsetup` command can be used with docker services as well as without docker (host setup). What do you think @bjester?

Reply:
Yeah I think that'll work!

Reply:
@bjester updated PR as per above.